cli/azd/cmd/util.go

```diff
@@ -283,7 +285,7 @@ func ensureEnvironmentInitialized(
 }
 
 func getSubscriptionOptions(ctx context.Context) ([]string, string, error) {
-	accountManager := account.NewManager(ctx)
+	accountManager := account.NewManager(config.GetConfig(), azcli.GetAzCli(ctx))
```

Comment:
Can we avoid loading the config file multiple times? In `promptLocation` and in `getSubscriptionOptions`

Reply:
Not right now without storing the config in context or without some major refactoring around these `util` methods.